torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1823,6 +1823,18 @@ def call_compiled_backward():
                     )
                     return tuple(out)
 
+                # Backward with forward inputs mutations is not supported in double backward.
+                if torch.is_grad_enabled() and any(
```

Comment:
this still has the runtime performance issues from before: we need to loop through the entire list at runtime (where # entries scales with number of inputs to the backward).

I would just make a new field on `ViewAndMutationMeta` that is something like `indices_of_inputs_that_require_grad_with_mutations_in_bw: List[int]`, that only contains the indices of inputs that we might need to error on